community/how-to-release.md

```diff
@@ -118,6 +118,8 @@ Uploading takes about one minute, after that, you can check it by your email at
 The svn repository of the DEV branch is: https://dist.apache.org/repos/dist/dev/inlong
 
 The svn repository of the Release branch is: https://dist.apache.org/repos/dist/release/inlong
+
+Please ensure that the key you add to dev and release branch are the same.
```

Comment:
->
Please ensure the `key` you added to the `dev` and `release` branches is the same.